BuildLoader.py

```diff
@@ -220,6 +221,12 @@ def __init__(self, *args, **kwargs):
 		self.PAYLOAD_LOAD_BASE     = 0
 		self.FWUPDATE_LOAD_BASE    = 0
 
+		self.PAYLOAD_FD_BASE       = 0x00B00000
```

Comment:
I don't believe you need these here.   It can be set in BoardConfig.py

Reply:
These values will be used inside of the FDF, if I don't give a default value in BuildLoader.py it is failing on APL build. Should default values be needed in this case or no?

Reply:
Maybe you should use   PAYLOAD_EXE_BASE  ?

Reply:
I will clean these up some more next week; I've pushed some new code but haven't fixed this yet.